stix_shifter_modules/guardium/configuration/config.json

```diff
@@ -1,19 +1,23 @@
 {
     "connection": {
         "type": {
-            "displayName": "IBM® Guardium"
+            "default": "guardium"
         },
         "host": {
             "type": "text",
-            "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9_:/\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9_:/\\-]*[A-Za-z0-9])$"
```

Comment:
Was there a reason this regex was changed? The other connectors are using it with the underscores.